chain/store/store.go

```diff
@@ -52,6 +52,29 @@ func NewLedgerStore() (*ChainStore, error) {
 	return chain, nil
 }
 
+func (cs *ChainStore) FastSyncDone(syncRootHash common.Uint256, fastSyncHeight uint32) {
+	sdb, err := NewStateDB(syncRootHash, cs)
+	if err != nil {
+		log.Fatalf("Error create new stateDB: %v", err)
+	}
+
+	err = sdb.trie.TryTraverse()
+	if err != nil {
+		log.Fatal("Trie traverse error: %v", err)
+	}
+
+	hs := sdb.trie.Hash()
+	if syncRootHash.CompareTo(hs) != 0 {
+		log.Fatal("State root not equal, should be: %v, we got: %v", syncRootHash.ToHexString(), hs.ToHexString())
+	}
+	cs.States = sdb
+
+	err = cs.persistSyncRootHeight(fastSyncHeight)
```

Comment:
Do you need to put a few other db items like `CurrentStateTrie`, `CurrentBlockHashKey`, etc? See logic in `persist()` function.